policy.php

```diff
@@ -0,0 +1,32 @@
+<?php
+	require_once("template.php");
+	require_once("config.php");
+	Template::render_header("Privacy Policy");
+
+	echo "<h1>" . _("Contact & Privacy Policy") . "</h2>";
+	echo "<h2>" . _("Contact") . "</h2>";
+	echo "##name## <br>";
+	echo "##email## <br>";
+	echo '<h2>' . _("Privacy Policy") . '</h2>';
+	echo '<h3>' . _("General") . "</h3>";
+	echo _("General
+	Based on article 13 of the Swiss Federal Constitution and the and the data protection provisions of the Swiss Confederation 
```

Comment:
This might be probably replaced by something like "Based on regulation (EU) 2016/679 of the European Parliament and of the Council of 27 April 2016 on the protection of natural persons with regard to the processing of personal data and on the free movement of such data" as it is more general.

Reply:
Under "General" should be placed:
- Who are we? (title)
- What personal data we collect and why we collect it (title)
-- Global: (sub title)
-- Telegram: (Sub title)

I could provide an example of a Privacy Policy I had to write for myself if you want this.
It should be compatible with the thingie you mentioned above.